libraries/core/src/main/kotlin/com/bumble/appyx/core/node/ParentNode.kt

```diff
@@ -74,6 +76,15 @@ abstract class ParentNode<NavTarget : Any>(
         manageTransitions()
     }
 
+    @Composable
+    final override fun DerivedSetup(innerContent: @Composable () -> Unit) {
+        CompositionLocalProvider(
+            LocalParentNodeMovableContent provides mutableMapOf()
```

Comment:
Should we create the `mutableMapOf` for every `ParentNode`? What if the `LocalParentNodeMovableContent` is set alongside `LocalSharedElementScope`? Is it possible? What are the implications?

This would effectively create a new map for every single `ParentNode` we have in the hierarchy, even if the Map isn't used

Reply:
Moved to Children  composable and add it when movableContent is enabled